### PR TITLE
add descriptions of various mailing lists

### PR DESCRIPTION
**Pull request by ajm188**
very open to feedback here.

bringing in at least @timtadh @matthewbentley @brenns10 @k4therin2 for discussion

### CODE REVIEW

File: organizational/communications.rst [hunk 1]
```diff
@@ -19,22 +19,50 @@ Email
...
+
+      This list is moderated.
     - `hackers-summer mailing list
       <https://lists.case.edu/wws/info/hackers-summer>`_
+      For discussion among Hacker Society members that are in the Cleveland
+      area over the summer.
+
+      This list is probably not moderated.
     - `acm-announce mailing list
       <https://lists.case.edu/wws/info/hackers-summer>`_
```
Comment (mtbentley) [inline organizational/communications.rst]: Oops

Comment (ajm188) [inline organizational/communications.rst]: lol. `git blame`

**Comment by mtbentley** [comment]
Add hacsoc-officers

File: organizational/communications.rst [hunk 2]
```diff
@@ -19,22 +19,50 @@ Email
...
+      For announcements about server downtime. If you have a service running on
+      our servers, you should be on this list.
+
+      This list is moderated.
     - `hackers-summer mailing list
       <https://lists.case.edu/wws/info/hackers-summer>`_
+      For discussion among Hacker Society members that are in the Cleveland
+      area over the summer.
+
+      This list is probably not moderated.
```
Comment (zwh2) [inline organizational/communications.rst:56]: Can we figure out if this is or isn't moderated.

**Comment by zwh2** [comment]
Can we just delete all of lists that are "a relic from an earlier time"

File: organizational/communications.rst [hunk 3]
```diff
@@ -19,22 +19,56 @@ Email
...
       <https://lists.case.edu/wws/info/acm-members>`_
+      A relic from an earlier time.
     - `acm-officers mailing list
       <https://lists.case.edu/wws/info/acm-officers>`_
+      A relic from an earlier time.
     - `acm-talk mailing list
       <https://lists.case.edu/wws/info/acm-talk>`_
+      A relic from an earlier time.
 
 - Has to move over to some Google hosted service before the end of 2015?
```
Comment (thomasrussellmurphy) [inline organizational/communications.rst]: Change this to "the most used lists will be migrated to Google Groups and others may be deleted"?

**Review by thomasrussellmurphy — CHANGES_REQUESTED** [review]
While this may need more work, having a good place to start from would be great, particularly since this hasn't gotten very far. @ajm188 can you address talking about Google Groups in the future, then see if we can merge this?

File: organizational/communications.rst (hunk 3 above)
Comment (ajm188) [reply organizational/communications.rst]: Yeah I'll update in a sec. Boy howdy, I wish you could "request changes" as a diff, and then I could click a button that applied them... 😛

**Review by thomasrussellmurphy — APPROVED** [review]

**Comment by thomasrussellmurphy** [comment]
Features! I'm going to merge this.

**Comment by ajm188** [comment]
beat ya :)

**Comment by thomasrussellmurphy** [comment]
Aww. I was going to try squash+merge, but too late.

**Comment by ajm188** [comment]
Call me old fashioned, but I like my `--no-ff` merges

**Pull request merged**